### DISCUSSION TITLE
2023년 03월 17일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#9] [General]
# 2023년 03월 17일

LOG 오늘은 놀기로 한 날이라서 한강에 놀러갔다. 날씨가 따뜻했는데 바람이 많이 불어서 산발이 되었다.. ㅎ 레벨1 글쓰기를 했다. 오랜만에 글을 써서 그런가 정말 안 써졌다. 궁금한 사람은 초안을 볼 수 있다 -> [ level1-writing …

https://cyma-s.github.io/2023-03-17/

**Comment by drunkenhw** [comment]
2판이에요

**Reply by Cyma-s** [reply]
다시 생각해보니 3판인 것 같네요
